build/tools/esp32/esptool_py/Makefile

```diff
@@ -76,6 +78,14 @@ ALL:flash
 
 APP:app-flash
 
+ROMFS:romfs
```

Comment:
is it necessary to limit this romfs burning under the CONFIG_FS_ROMFS?

Reply:
if disable CONFIG_FS_ROMFS, this function will fail definitely.